sogs/routes/rooms.py

```diff
@@ -960,3 +960,27 @@ def delete_all_posts(room, sid):
     if not deleted:
         abort(http.NOT_FOUND)
     return jsonify({})
+
+@rooms.delete("/room/all/<SessionID:sid>")
```

Comment:
`/room/ANYTHING/...` is reserved for specific access into a single room (so that if someone wants to create a room with id "all" it wouldn't conflict with the endpoint added here).  There is, however, the `/rooms` prefix already that we could add onto (currently it only has the top-level target to return a list of all server rooms) by renaming this endpoint to `/rooms/all/<SessionID:sid>`

Reply:
Also: the non-existent `/user/.../deleteMessages` endpoint mentioned in routes/users.py can get updated to match this new endpoint.

Reply:
Thanks. Changed as described.